README.md

```diff
@@ -10,6 +10,9 @@ Documentation on FC can be found [here](https://chaste.cs.ox.ac.uk/trac/wiki/Fun
 
 An ongoing attempt to document the Web Lab and all its interconnected technologies can be viewed in the [weblab docs repository](https://github.com/ModellingWebLab/weblab_docs).
 
+## Linux-only
```

Comment:
It does work on Mac OS though :)

Reply:
That's also linux :D

Reply:
Not really. It's Unix-based, but more like BSD than Linux iirc.